test/CMakeLists.txt

```diff
@@ -14,8 +14,8 @@ add_lit_testsuite(test-all "Run all the tests"
 
 add_lit_testsuite(test-fat-ptr "Run fat ptr tests"
   -v
-  ${CMAKE_CURRENT_SOURCE_DIR}/fat_ptr
-  ARGS
+        ${CMAKE_CURRENT_SOURCE_DIR}/fat_ptr_symbolic
+        ARGS
```

Comment:
formatting. 

this replaced fat ptr tests but does not update text description. what about old tests this replaced?

Reply:
Thanks for catching this! Added new fatptr-symbolic target. Fixed formatting.